source/sas_compress.cpp

```diff
@@ -83,9 +83,6 @@ class LZ4Compressor : public SAS::Compressor
   static SAS::Compressor* get();
 
 private:
-  // The maximum size of input we can process in one go.
-  static const int MAX_INPUT_SIZE = 4096;
```

Comment:
Because we no longer have a fixed max input size - instead, if the size we can cope with (_buffer_len) is too small, we double it and try again.